apps/frontend/src/modules/projects/slices/project.slice.ts

```diff
@@ -4,20 +4,33 @@ import { DataStatus } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
 import { type ProjectGetAllItemResponseDto } from "~/modules/projects/projects.js";
 
-import { loadAll } from "./actions.js";
+import { getById, loadAll } from "./actions.js";
 
 type State = {
 	dataStatus: ValueOf<typeof DataStatus>;
+	project: null | ProjectGetAllItemResponseDto;
 	projects: ProjectGetAllItemResponseDto[];
 };
 
 const initialState: State = {
 	dataStatus: DataStatus.IDLE,
+	project: null,
 	projects: [],
 };
 
 const { actions, name, reducer } = createSlice({
 	extraReducers(builder) {
+		builder.addCase(getById.pending, (state) => {
+			state.dataStatus = DataStatus.PENDING;
```

Comment:
Let's use dataStatus for get all, add a separate property for project status, like `projectStatus`